src/main/java/step5/domain/MoveCount.java

```diff
@@ -0,0 +1,20 @@
+package step5.domain;
+
+public class MoveCount {
+    private int value;
+
+    public MoveCount() {
+    }
```

Comment:
주 생성자를 호출하는 방식으로 변경해보면 어떨까요?
```suggestion
    public MoveCount() {
        this(0);
    }
```